README.md

```diff
@@ -18,6 +18,8 @@ Quick links to our resources are:
 
 ## Installation
 
+How do I install Newscoop? [https://github.com/sourcefabric/Newscoop/tree/master/newscoop/docs]
```

Comment:
It should be `How do I install Newscoop? [Check here](https://github.com/sourcefabric/Newscoop/tree/master/newscoop/docs).`